proto/sf/metering/v1/metering.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package sf.metering.v1;
+
+option go_package = "github.com/streamingfast/dmetering/pb/sf/metering/v1;pbmetering";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+service Metering {
+  rpc Emit(Event) returns (google.protobuf.Empty) {}
+}
+
+message Event {
+  string user_id = 1;
+  string api_key_id  = 2;
+  string ip_address = 3;
+
+  // Defines the service that emitted the event (sf.firehose.v1/Blocks ...)
+  string endpoint = 4;
```

Comment:
I know you did not choose the name ourself, but I'm wondering if this should not be called `service`, at least the documentation seems to say so ...